workers/unity/Packages/com.improbable.gdk.tools/RedirectedProcess.cs

```diff
@@ -1,13 +1,22 @@
 using System;
+using System.Collections.Generic;
 using System.Diagnostics;
 using System.IO;
 using System.Text;
+using System.Threading.Tasks;
 using Improbable.Gdk.Tools.MiniJSON;
 using UnityEngine;
 using Debug = UnityEngine.Debug;
 
 namespace Improbable.Gdk.Tools
 {
+    public class RedirectedProcessResult
+    {
+        public int ExitCode;
+        public List<string> Stdout;
```

Comment:
nit: Why a list over a Stack?

Reply:
No reason. I'll change it.

Reply:
Wait, this probably shouldn't be a stack. FILO semantics don't really make sense for log messages.